api/views.py

```diff
@@ -0,0 +1,9 @@
+from django.shortcuts import redirect
+
+
+def view_404(request):
+    return redirect('/')
+
+
+def view_home(request):
+    return redirect('http://project-travel-mate.github.io/Travel-Mate/')
```

Comment:
Move this URL string to a new file `nomad/constants.py` and name the variable to `URL_NOT_FOUND_REDIRECT`